docs/mirror_configuration.md

```diff
@@ -217,3 +219,13 @@ Example:
 [mirror]
 proxy=http://myproxy.com
 ```
+
+## download-mirror
+By default bandersnatch downloads packages from the URL supplied in the master server server's json response. This option asks bandersnatch to try to download from the configured PyPI mirror first, and fallback to the URL supplied by the master server if it was not successful (unable to get content or checksum mismatch).
```

Comment:
```suggestion
## download-mirror

By default bandersnatch downloads packages from the URL supplied in the master server server's json response.
This option asks bandersnatch to try to download from the configured PyPI mirror first, and fallback to the
URL supplied by the master server if it was not successful (unable to get content or checksum mismatch).
```
Just good markdown form to add newlines here ... I thought we had prettier running here - must not ...